downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc

```diff
@@ -38,7 +39,7 @@ registry_password='<registry password>'
 # Automation Hub Configuration
 #
 automationhub_admin_password='<password>'
-automationhub_pg_host='database-01.acme.org'
+automationhub_pg_host='tower.acme.org'
```

Comment:
https://github.com/adonisgarciac

Can you comment?

Reply:
I think that it would be a good idea to change it. But, it is not only necessary change it in automationhub_pg_host. If we want "upgrade" the example to the current naming (tower to controller) we need change it in more than one place:

https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/blame/main/downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc#L21
https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/blame/main/downstream/modules/platform/ref-platform-non-inst-database-inventory.adoc#L42
https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/blame/main/downstream/modules/platform/ref-example-platform-ext-database-inventory.adoc#L20

Reply:
https://issues.redhat.com/browse/AAP-4728 Created to deal with this change.